Server/mods/deathmatch/logic/packets/CEntityAddPacket.cpp

```diff
@@ -978,6 +978,42 @@ bool CEntityAddPacket::Write(NetBitStreamInterface& BitStream) const
                         BitStream.Write(currentWeaponSlot);
                     }
 
+                    // Animation
+                    if (BitStream.Can(eBitStreamVersion::AnimationsSync))
+                    {
+                        const SPlayerAnimData& animData = pPed->GetAnimationData();
+
+                        // Contains animation data?
+                        bool animRunning = !animData.blockName.empty() && !animData.animName.empty();
```

Comment:
Nit-picking here, but you could move this decision logic, whether an animation is running or not, to the SPlayerAnimData class: `animData.IsAnimating()`